### ISSUE TITLE
Code the new Lockwise product page at www.mozilla.org/firefox/lockwise

### ISSUE THREAD

**Issue by ejregithub** [mozilla/bedrock#7570]
Description: Create the product page for Lockwise

----

***Success Criteria***
- [ ] Page is live on or before October 22nd to Tier One locales

***Risks***

***Tasks***
- [x] Code the page
- [x] Add conditional content for the Add-on install button for dekstop
- [ ] Find all links on www.mozilla.org that point at lockwise.firefox.com and point at the new page. **NOTE** Remove same-site utm parameters. If we previously included utm params on lockwise.firefox.com, we should not include them when we change the link to www.mozilla.org/*.
- [ ] Set up a 301 redirect lockwise.firefox.com to www.mozilla.org/firefox/lockwise
- [x] Source [localized images](https://drive.google.com/drive/folders/19Zrt7jc11e25PmSydNzsRkkJOzo6vLHL), code in the page
- [x] Source Adjust links, code in the page
- [x] Extract strings for L10n w. Tier One as MVP - but goal to make it available to all locales (which implies extracting strings at least two weeks prior to the target launch date)
- [x] Identify Protocol component requirements. Build / update new components that are required.
- [x] Wrap the addon CTA in conditional logic to work only for Fx 69 and below
- [x] Link to the product FAQ that is moving to SUMO

**Comment by rraue** [comment]
Its very important that we redirect 301 from lockwise.firefox.com to www. Could you concretise this in the task @ejregithub please.

**Comment by ejregithub** [comment]
> Its very important that we redirect 301 from lockwise.firefox.com to [www](http://www). Could you concretise this in the task @ejregithub please.

@rraue - that 301 redirect requirement is now explicit in the tasks.

**Comment by Errolyn** [comment]
Assuming it makes sense for me to do so, I would really like to work on this.

**Comment by Errolyn** [comment]
Blocked and waiting for the design to be done/finalized.

**Comment by Errolyn** [comment]
I can get started on the page, still need a few more assets but I have enough to get moving on.

**Comment by hoosteeno** [comment]
@Errolyn can you update on status of the code here?

**Comment by Errolyn** [comment]
Code is done and ready for review, I have a couple questions on some of the other elements in this issue and if they are things I should be taking on? 

> Find all links on www.mozilla.org that point at lockwise.firefox.com and point at the new page

this seems like something I should do now that the page is getting reviewed in a separate pr?

>Set up a 301 redirect lockwise.firefox.com to www.mozilla.org/firefox/lockwise

I'm not sure if I have the access to do this? should I look into how or is someone else already waiting on the changes to create the 301 redirect?

> Link to the product FAQ that is moving to SUMO

We don't have a "FAQ" link on the page in the design so I assume this isn't something we need anymore?

>Source Adjust links, code in the page

I'm not sure if there is anything else that needs to be done with with content but everything is up to date with what I have been given in the design and copy.

**Comment by Errolyn** [comment]
https://bedrock-demo-lockwise.oregon-b.moz.works/en-US/firefox/lockwise/ --  is the demo, and is ready for stakeholder review @hoosteeno

**Comment by hoosteeno** [comment]
> > Find all links on [www.mozilla.org](http://www.mozilla.org) that point at lockwise.firefox.com and point at the new page
> 
> this seems like something I should do now that the page is getting reviewed in a separate pr?
> 

Agree.

> > Set up a 301 redirect lockwise.firefox.com to [www.mozilla.org/firefox/lockwise](http://www.mozilla.org/firefox/lockwise)
> 
> I'm not sure if I have the access to do this? should I look into how or is someone else already waiting on the changes to create the 301 redirect?

Let's wait until the page is live, then file a bug.

> 
> > Link to the product FAQ that is moving to SUMO
> 
> We don't have a "FAQ" link on the page in the design so I assume this isn't something we need anymore?

Right -- the 'Support' link does this work now.

> 
> > Source Adjust links, code in the page
> 
> I'm not sure if there is anything else that needs to be done with with content but everything is up to date with what I have been given in the design and copy.

I don't think the adjust link is correct. I've asked Loren to help.

**Comment by hoosteeno** [comment]
In the slack channel and in email, I sent out a request for final feedback on this built page. The feedback document is open to Mozilla staff, and is [here](https://docs.google.com/document/d/1tHpbhsr0YS2RiiBgw1QcgpxN2t0XAErpGx26ZfHTPko/edit#).

**Comment by Errolyn** [comment]
Just a heads up The ff 70+ CTA button will not work for you unless you do the following steps (this will not be an issue when the page is up on the Mozilla domain)

1. type `about:config` into a new browser tab
2. search for `browser.uitour.testingOrigins`
3. if it doesn’t exist add `browser.uitour.testingOrigins` as a string input
4. give that the value `https://bedrock-demo-lockwise.oregon-b.moz.works`

The button should then be able to open up the preferences menu and highlight Logins and Passwords.

UiTour is only enabled for mozilla.org, and to test it on the demo branch you will need to manually add the domains that are ok.

**Comment by ejregithub** [comment]
This is live, closing
